mmv1/products/containerattached/api.yaml

```
@@ -0,0 +1,273 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: ContainerAttached
+display_name: Anthos Attached clusters
+versions:
+  - !ruby/object:Api::Product::Version
+    name: ga
+    base_url: https://{{location}}-gkemulticloud.googleapis.com/v1/
+scopes:
+  - https://www.googleapis.com/auth/cloud-platform
+objects:
+  - !ruby/object:Api::Resource
+    name: 'Cluster'
+    base_url: projects/{{project}}/locations/{{location}}/attachedClusters
+    create_url: projects/{{project}}/locations/{{location}}/attachedClusters?attached_cluster_id={{name}}
+    delete_url: projects/{{project}}/locations/{{location}}/attachedClusters/{{name}}
+    update_url: projects/{{project}}/locations/{{location}}/attachedClusters/{{name}}
+    self_link: projects/{{project}}/locations/{{location}}/attachedClusters/{{name}}
+    update_verb: :PATCH
+    update_mask: true
+    delete_verb: :DELETE
+    description: |
+      An Anthos attached cluster.
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'API reference': 'https://cloud.google.com/anthos/clusters/docs/multi-cloud/reference/rest/v1/projects.locations.attachedClusters'
+        'Multicloud overview': 'https://cloud.google.com/anthos/clusters/docs/multi-cloud'
+      api: 'https://cloud.google.com/anthos/clusters/docs/multi-cloud/reference/rest'
+    async: !ruby/object:Api::OpAsync
+      operation: !ruby/object:Api::OpAsync::Operation
+        path: 'name'
+        full_url: 'https://{{location}}-gkemulticloud.googleapis.com/v1/{{op_id}}'
+        wait_ms: 1000
+      result: !ruby/object:Api::OpAsync::Result
+        path: 'response'
+        resource_inside_response: true
+      status: !ruby/object:Api::OpAsync::Status
+        path: 'done'
+        complete: 'true'
+        allowed:
+          - 'true'
+          - 'false'
+      error: !ruby/object:Api::OpAsync::Error
+        path: 'error/errors'
+        message: 'message'
+    properties:
+      - !ruby/object:Api::Type::KeyValuePairs
+        name: annotations
+        description: |
+          Optional. Annotations on the cluster. This field has the same
+          restrictions as Kubernetes annotations. The total size of all keys and
+          values combined is limited to 256k. Key can have 2 segments: prefix (optional)
+          and name (required), separated by a slash (/). Prefix must be a DNS subdomain.
+          Name must be 63 characters or less, begin and end with alphanumerics,
+          with dashes (-), underscores (_), dots (.), and alphanumerics between.
+      - !ruby/object:Api::Type::NestedObject
+        name: authorization
+        description: |
+          Configuration related to the cluster RBAC settings.
+        properties:
+          - !ruby/object:Api::Type::Array
+            name: admin_users
```

Fields in api.yaml use camelCase - it's probably not a big deal since this isn't a real API field but for consistency.

```suggestion
            name: adminUsers
```

Using this field name with custom expand & flatten is consistent with the aws and azure clusters.